desktop/apps/consignments2/components/upload_photo/index.js

```diff
@@ -24,76 +24,76 @@ function UploadPhoto (props) {
 
   return (
     <div className={b()}>
-      <label
-        htmlFor='file'
-        className={b('drop-area')}
-        onDragOver={(e) => e.preventDefault()}
-        onDrop={(e) => {
-          e.preventDefault()
-          selectPhotoAction(e.dataTransfer.files[0])
-        }}
-      >
-        <div className={b('drop-area-contents')}>
-          <input
-            type='file'
-            name='file'
-            id='file'
-            className={b('file-upload')}
-            onChange={(e) => selectPhotoAction(e.target.files[0])}
-          />
-          <div className={b('camera-icon')}>
-            <Camera />
-          </div>
-          <div className={b('cta')}>
-            Drag or Click to upload photos
-          </div>
-        </div>
-      </label>
-      <div className={b('upload-instructions')}>
-        Please upload JPG or PNG image files 1000x1000 pixels or more. Image files should have a file size less than 30mb.
-      </div>
-      <CheckboxInput
-        item={'skip'}
-        label={'No photo currently available'}
-        onChange={updateSkipPhotoSubmissionAction}
-        value={skipPhotoSubmission}
-      />
-      {
-        uploadedImages.map((file, index) =>
-          <UploadedImage file={file} key={`${file.fileName}-${index}`} />
-        )
-      }
-      <div
-        className={b('submit-button').mix('avant-garde-button-black')}
-        onClick={submitPhotoAction}
-        disabled={!nextEnabled}
-      >
-        Submit
+      <div className={b('title')}>
```

Comment:
In this case I also just added a title and nested everything else (these big diffs are misleading).